code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -281,3 +281,11 @@
 	name = "\improper Anti-Plague Sequence Omega autoinjector"
 	desc = "A small autoinjector containing 15 units of Anti-Plague Sequence Omega.  Cures all cases of the Necrotizing Plague. Also heals dead limbs."
 	list_reagents = list("zombiecure4" = 15)
+	
+/obj/item/reagent_containers/hypospray/autoinjector/hyper_medipen
+	name = "\improper suspicious medipen"
```

Comment:
```suggestion
	name = "suspicious medipen"
```

if we're not capitalizing any letters, the improper tag isn't needed.

either do `name = "\improper Suspicious Medipen"` or the above suggested change.